deployment/components/topology-updater/topologyupdater-mounts.yaml

```diff
@@ -17,8 +17,9 @@
 - op: add
   path: /spec/template/spec/containers/0/volumeMounts
   value:
-  - name: kubelet-podresources-conf
-    mountPath: /host-var/lib/kubelet/config.yaml
```

Comment:
Why are we dropping this? I think kubelet config is unrelated to the reactive updates. We might still use this is configz is not used

Reply:
Because it goes under the `/host-var/lib/kubelet` so we could access it anyway

Reply:
But the problem is that the kubelet config does not necessarily reside under `/var/lib/kubelet`.

Reply:
I see. OK let me fix that

Reply:
Done.